hpack/definitions.yaml

```diff
@@ -0,0 +1,91 @@
+# This file defines sensible defaults but does not actually apply
+# any of them, so it can be safely included.
+
+_definitions:
+  _top-level:
+    - &meta
+        version:            '0.2.0.0'
+        description:        'Please see the README on Github at <https://github.com/serokell/ariadne#readme>'
+        homepage:           'https://github.com/serokell/ariadne#readme'
+        bug-reports:        https://github.com/serokell/ariadne/issues
+        author:             Serokell
+        maintainer:         Serokell <hi@serokell.io>
+        copyright:          2018 Serokell
+        license:            MPL-2.0
+        license-file:       COPYING.md
+        github:             serokell/ariadne
+        extra-source-files: []
+
+    - &default-extensions
+        - ApplicativeDo
+        - BangPatterns
+        - ConstraintKinds
+        - DataKinds
+        - DeriveFoldable
+        - DeriveFunctor
+        - DeriveGeneric
+        - DeriveTraversable
+        - EmptyCase
+        - FlexibleContexts
+        - FlexibleInstances
+        - GADTs
+        - GeneralizedNewtypeDeriving
+        - LambdaCase
+        - MonadFailDesugaring
+        - MultiParamTypeClasses
+        - MultiWayIf
+        - NegativeLiterals
+        - NoImplicitPrelude
+        - OverloadedLabels
+        - OverloadedStrings
+        - PatternSynonyms
+        - PolyKinds
+        - RankNTypes
+        - RecordWildCards
+        - RecursiveDo
+        - ScopedTypeVariables
+        - StandaloneDeriving
+        - TemplateHaskell
+        - TupleSections
+        - TypeApplications
+        - TypeFamilies
+        - TypeOperators
+        - UndecidableInstances
+        - ViewPatterns
+
+    - &dependencies
+        - base >= 4.7 && < 5
+        - universum
```

Comment:
AFAIU, because of that `knit` depends on `universum` and I suspect @int-index may not like it (beside not liking `hpack` in general :sweat_smile:).

Reply:
That one is fixed in the next PR, will probably port it to this one

Reply:
Fixed with an override in the local `package.yaml`